Zend/zend_compile.c

```diff
@@ -7662,8 +7662,7 @@ void zend_compile_class_decl(znode *result, zend_ast *ast, bool toplevel) /* {{{
 
 	if (UNEXPECTED((decl->flags & ZEND_ACC_ANON_CLASS))) {
 		/* Serialization is not supported for anonymous classes */
-		ce->serialize = zend_class_serialize_deny;
```

Comment:
I think it'd be safer to add ZEND_ACC_NOT_SERIALIZABLE but keep zend_class_serialize_deny until the next minor release, especially with the feature freeze in a few days.

Older releases of msgpack and igbinary (and possibly others) would depend on zend_class_serialize_deny and zend_class_unserialize_deny to work correctly, and some of them would have releases that would compile in 8.1.0+ stable or be used to beta test 8.1

(e.g. to avoid unserializing an invalid Closure)

Reply:
@krakjoe Thoughts on this? I'd rather not duplicate the info, and we're still half a year away from the release.

Reply:
I don't like duplication either, and I don't like leaving broken API exposed for longer than we must.